x-pack/plugins/security_solution/public/management/cypress/cypress.config.ts

```diff
@@ -41,6 +41,10 @@ export default defineCypressConfig({
     KIBANA_PASSWORD: 'changeme',
     ELASTICSEARCH_USERNAME: 'system_indices_superuser',
     ELASTICSEARCH_PASSWORD: 'changeme',
+
+    grepFilterSpecs: true,
```

Comment:
A bit strange (to me) to have `env` variables in lower case and not upper case and snake_cased.

Reply:
understandable. although these variables come from [cypress grep](https://www.npmjs.com/package/@cypress/grep), so I'd just leave them as they are.